e107_handlers/bbcode_handler.php

```diff
@@ -574,7 +575,8 @@ function renderButtons($template='', $id='', $options=array())
 		$temp['maintenance']= $BBCODE_TEMPLATE_ADMIN;
 		$temp['comment'] 	= $BBCODE_TEMPLATE_COMMENT;
 		$temp['signature'] 	= $BBCODE_TEMPLATE_SIGNATURE;
-	
+		$temp['forum'] 	    = $BBCODE_TEMPLATE_FORUM;
+
```

Comment:
Here we need a v2.x template format and to load the forum template from the Forum plugin. (not the core). 
Let's discuss further on Gitter.

Reply:
You removed e_bb.php functionality with this
https://github.com/e107inc/e107/issues/3394